src/Plugin/EPoetryLogPlugin.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Plugin;
+
+use Phpro\SoapClient\Event\FaultEvent;
+use Phpro\SoapClient\Event\RequestEvent;
+use Phpro\SoapClient\Event\ResponseEvent;
+use Phpro\SoapClient\Events;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class ePoetryLogPlugin
+ */
+class EPoetryLogPlugin implements EventSubscriberInterface
```

Comment:
We can remove the `EPoetry` part from `EPoetryLogPlugin` since it's already in the namespace. Let's call this class `LoggerSubscriber` or something of sort, to stress it's an event subscriber.